### PR TITLE
NOISSUE: Add Documentation for Running Algo with Args

### PR DESCRIPTION
**Pull request by WashingtonKK**
# What type of PR is this?

This is an enhancement since it adds documentation for running an algo that requires command line args.

## What does this do?

It adds documentation for running an algo that requires command line args.

## Which issue(s) does this PR fix/relate to?

- No issue related

## Have you included tests for your changes?

Not needed for docs.

## Did you document any new/modified features?

Yes, this adds documentation for a feature in cocos.

### CODE REVIEW

File: docs/algorithms.md [hunk 1]
```diff
@@ -477,6 +477,16 @@ Terminal recording session
...
 
+## Running Python Algorithms with arguments
+
+To run a python algo that requires command line arguments, you can append the algo command on cli with the arguments needed as shown in the addition example:
+
+```bash
+./build/cocos-cli algo ./test/manual/algo/addition.py ./private.pem -a python --args="--a" --args="100" --args="--b" --args="20"
+```
+
+Details of how to run the full addition example with args can be found [here](https://github.com/ultravioletrs/cocos/blob/7a2789fb5fda48282ef0c1d516aa8ba36421f5f1/test/manual/algo/README.md).
```
Comment (rodneyosodo) [inline docs/algorithms.md:488]: Port this documentation to here since this is the main documentation place

**Review by rodneyosodo — CHANGES_REQUESTED** [review]

File: docs/algorithms.md [hunk 2]
```diff
@@ -477,6 +477,16 @@ Terminal recording session
 
 For real-world examples to test with cocos, see our [AI repository](https://github.com/ultravioletrs/ai).
 
+## Running Python Algorithms with arguments
+
+To run a python algo that requires command line arguments, you can append the algo command on cli with the arguments needed as shown in the addition example:
```
Comment (SammyOina) [inline docs/algorithms.md:482]: not only python arguments are for all algos

File: docs/algorithms.md [hunk 3]
```diff
@@ -477,6 +477,16 @@ Terminal recording session
...
+
+```bash
+./build/cocos-cli algo ./test/manual/algo/addition.py ./private.pem -a python --args="--a" --args="100" --args="--b" --args="20"
```
Comment (SammyOina) [inline docs/algorithms.md:485]: include more detail on how to add arguments, order, how may you need to add, show how the added args will be passed by agent to the algo

**Review by SammyOina — CHANGES_REQUESTED** [review]

File: docs/algorithms.md [hunk 4]
```diff
@@ -382,10 +382,10 @@ MANAGER_QEMU_OVMF_VARS_FILE=/usr/share/edk2/x64/OVMF_VARS.fd \
 go run main.go
 ```
 
-Export the agent grpc url from computation server logs
+Export the agent grpc url from computation server logs, by default
```
Comment (SammyOina) [inline docs/algorithms.md:385]: this sentence sounds incomplete at the end

File: docs/algorithms.md [hunk 5]
```diff
@@ -477,6 +477,65 @@ Terminal recording session
 
 For real-world examples to test with cocos, see our [AI repository](https://github.com/ultravioletrs/ai).
 
+## Running Algorithms with arguments
+
+To run an algo that requires command line arguments, you can append the algo command on cli with the arguments needed as shown in the addition example, which we will run with args below:
```
Comment (SammyOina) [inline docs/algorithms.md:482]: algorithm

**Review by SammyOina — CHANGES_REQUESTED** [review]

**Review by smithjilks — APPROVED** [review]
LGTM!

**Review by SammyOina — CHANGES_REQUESTED** [review]

**Review by SammyOina — APPROVED** [review]

**Pull request merged**